r10k.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'multi_json', '~> 1.10'
 
   s.add_dependency 'puppet_forge', '~> 2.2'
-  s.add_dependency 'semantic_puppet', '~> 0.1.0'
+  s.add_dependency 'semantic_puppet', '~> 1.0.0'
```

Comment:
I think this change will stop folks on puppet-agent 1.x from installing r10k, which would be too bad.
What if you changed the dependency to be `s.add_dependency 'semantic_puppet', '>= 0.1.0', < '2.0.0'`?